lib/callbacks.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+const fs = require('fs');
+const util = require('util');
+const faker = require('faker');
+let path = process.argv.slice(2)[0];
+let file = process.cwd()+`/data/${path}`;
+
+const readMyFile = function(file, cb) {
+  fs.readFile(file, 'utf8', function(err, content) {
+    if (err) return cb(err, null);
+    cb(null, content);
+    console.log(content);
+  });
+};
+
+readMyFile(file);
```

Comment:
You're calling `readMyFile` with the incorrect number of arguments, and thus your code is breaking. Again, we never want to call functions in our module file like this. Module files should export functions that we can manually call AFTER we `require` that module. 

<img width="949" alt="Screen Shot 2019-10-27 at 9 07 15 PM" src="https://user-images.githubusercontent.com/22336391/67651345-c494a100-f8fd-11e9-9cec-b8838a876373.png">